### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by cloud-gov-pages-operations**
## :robot: This is an automated release PR
chore: release 0.2.0
tag to create: 0.2.0
increment detected: MINOR

### Added

- Added site custom domains page | Updated domain model and admin | #4199

### Fixed

- Site serializer domain and branch config pick
- Fixed branchViewLink to use branch config and domain model #4032
- restore viewbox to svg icons for proper rendering

### Maintenance

- extend dependabot config to admin and metrics (#4217)
- add slack notification to release (#4216)
- update dependencies
- run dev tests in parallel with build, update dependabot (#4196)
- Build test webpack assets in non-version-controlled directory (#4191)
- add github release to production deploy (#4216)
- update deployment documentation
- correct release PR behavior (#4162)
## security considerations
Noted in individual PRs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+### Added
+
+- Added site custom domains page | Updated domain model and admin | #4199
+
+### Fixed
+
+- Site serializer domain and branch config pick
+- Fixed branchViewLink to use branch config and domain model #4032
+- restore viewbox to svg icons for proper rendering
```
Comment (svenaas) [inline CHANGELOG.md:11]:
```suggestion
- Restore viewbox to svg icons for proper rendering
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+
+### Maintenance
+
+- extend dependabot config to admin and metrics (#4217)
```
Comment (svenaas) [inline CHANGELOG.md:15]:
```suggestion
- Extend dependabot config to admin and metrics (#4217)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+- add slack notification to release (#4216)
```
Comment (svenaas) [inline CHANGELOG.md:16]:
```suggestion
- Add slack notification to release (#4216)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+- update dependencies
```
Comment (svenaas) [inline CHANGELOG.md:17]:
```suggestion
- Update dependencies
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+- run dev tests in parallel with build, update dependabot (#4196)
```
Comment (svenaas) [inline CHANGELOG.md:18]:
```suggestion
- Run dev tests in parallel with build, update dependabot (#4196)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+- Build test webpack assets in non-version-controlled directory (#4191)
+- add github release to production deploy (#4216)
```
Comment (svenaas) [inline CHANGELOG.md:20]:
```suggestion
- Add github release to production deploy (#4216)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,26 @@
...
+- update deployment documentation
```
Comment (svenaas) [inline CHANGELOG.md:21]:
```suggestion
- Update deployment documentation
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,26 @@
...
+- correct release PR behavior (#4162)
```
Comment (svenaas) [inline CHANGELOG.md:22]:
```suggestion
- Correct release PR behavior (#4162)
```

**Review by svenaas — CHANGES_REQUESTED** [review]
Inconsistent capitalization probably came straight from our commit messages. It's not terribly important, but it makes the changelog nicer if we fix it.

**Comment by drewbo** [comment]
@svenaas I'd like to avoid making any changes to the `release` branch directly and also avoid editing the current section of the CHANGELOG that is autogenerated. We can (1) try to standardize with automation and notes and (2) edit the changelog  and/or release afterwards

**Review by apburnes — APPROVED** [review]

**Review by svenaas — APPROVED** [review]
I'm good with this as it is.

**Pull request merged**